src/redux/categories/categories.js

```diff
@@ -0,0 +1,29 @@
+const ADDCATEGORY = 'ADD_CATEGORY';
+const REMOVECATEGORY = 'REMOVE_CATEGORY';
+const CHECKSTATUS = 'CHECK_STATUS';
```

Comment:
- [ ] As mentioned above the value of action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`.